src/types/index.tsx

```diff
@@ -0,0 +1,38 @@
+export interface Todo {
+  id: number,
+  userId: number,
+  title: string,
+  completed: boolean,
+  createdAt: string,
+  updatedAt: string,
+}
+
+export type Todos = {
+  todos: Todo[],
+};
+
+export interface User {
+  id?: number,
+  name?: string,
+  username?: string,
+  email?: string,
+  phone?: string,
+  website?: string,
+  createdAt?: string,
+  updatedAt?: string,
+}
+
+export type RootState = {
+  todos: Todo []|[];
```

Comment:
```suggestion
  todos: Todo[];
```
Пустой массив тоже подойдет в таком случае